src/internal.c

```diff
@@ -844,110 +850,400 @@ void SshResourceFree(WOLFSSH* ssh, void* heap)
 #endif
 }
 
-union wolfSSH_key {
+
+typedef struct WS_KeySignature {
+    byte keySigId;
+    word32 sigSz;
+    const char *name;
+    word32 nameSz;
+    union {
 #ifndef WOLFSSH_NO_RSA
-    RsaKey rsa;
+        struct {
+            RsaKey key;
+            byte e[256];
```

Comment:
this should be a max rsa macro instead of hard set

Reply:
This should be actual size based on the key. This value isn't needed until after we've read the key into the RsaKey.

Reply:
I double checked and these values weren't being used. Removed them.